wire-ios-data-model/Source/Model/UserClient/UserClient.swift

```diff
@@ -405,7 +405,8 @@ public extension UserClient {
         let model = payloadAsDictionary.optionalString(forKey: "model")?.removingExtremeCombiningCharacters
         let deviceClass = payloadAsDictionary.optionalString(forKey: "class")
         let activationDate = payloadAsDictionary.date(for: "time")
-
+        let mlsPublicKeys = payloadAsDictionary.dictionary(forKey: "mls_public_keys")
+        let mlsEd25519 = payloadAsDictionary.optionalString(forKey: "ed25519")
```

Comment:
Isn't this what you intended instead?
```suggestion
        let mlsPublicKeys = payloadAsDictionary.dictionary(forKey: "mls_public_keys")
        let mlsEd25519 = mlsPublicKeys.optionalString(forKey: "ed25519")
```

Reply:
yup.